models/src/main/kotlin/io/provenance/api/models/entity/EntityID.kt

```diff
@@ -0,0 +1,27 @@
+package io.provenance.api.models.entity
+
+import java.util.UUID
+
+sealed interface Entity { val id: String }
```

Comment:
Entities can be identified by the `x-uuid` header or Kong consumer headers via an API key